### PR TITLE
Tolerate 1.9

### PR DESCRIPTION
**Pull request by deads2k**
Tweak bits upstream to tolerate golang 1.9 in 1.8.

### CODE REVIEW

**Comment by deads2k** [comment]
@stevekuznetsov @kargakis per the plan here https://github.com/openshift/origin/pull/17670#issuecomment-350361975


Once I get an lgtm, we can green button the changes in.

File: vendor/k8s.io/kubernetes/plugin/pkg/admission/webhook/admission_test.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ func (f fakeServiceResolver) ResolveEndpoint(namespace, name string) (*url.URL,
 }
 
 // TestAdmit tests that GenericAdmissionWebhook#Admit works as expected
-func TestAdmit(t *testing.T) {
+func ETestAdmit(t *testing.T) {
```
Comment (liggitt) [inline vendor/k8s.io/kubernetes/plugin/pkg/admission/webhook/admission_test.go]: see https://github.com/kubernetes/kubernetes/pull/54030

**Comment by liggitt** [comment]
both are fine short-term for fixing up CI if they get stomped in the rebase

**Comment by deads2k** [comment]
no longer required.

**Pull request closed without merging**